cli/tests/pcluster/models/test_cluster.py

```diff
@@ -122,6 +122,25 @@ def test_tags(self, cluster, mocker, existing_tags):
             )
         ).is_true()
 
+        # Add name tag:
+        cluster_name = cluster.name
+        tags["parallelcluster:cluster-name"] = cluster_name
+        expected_tags_list = self._sort_tags(
+            [Tag(key=tag_name, value=tag_value) for tag_name, tag_value in tags.items()]
+        )
+
+        # Test method to add name tag
+        cluster._add_name_tag()
```

Comment:
This line can be moved to add together with cluster._add_version_tag()

Reply:
Done!